README.md

```diff
@@ -350,6 +376,50 @@ store.on('invalidate', () => {
 });
 ```
 
+### Connecting Store Updates To Widgets
+
+Store data can be connected to widgets within your application using the [Containers & Injectors Pattern](https://github.com/dojo/widget-core#containers--injectors) supported by `@dojo/widget-core`. `@dojo/stores` provides a specialized injector that invalidates store containers on two conditions:
+
+1. The recommended approach is to register `paths` on container creation to ensure invalidation will only occur when state you are interested in changes.
+2. A catch all when no `paths` are defined for the container, it will invalidate when any data changes in the store.
```

Comment:
Maybe change this sentence to be a bit more consistent with the previous one as a second condition?